### PR TITLE
NO3 & NP3 EntitySmallWaterDrop

### PR DESCRIPTION
**Pull request by sonicdcer**
https://decomp.me/scratch/Fw6kc

### CODE REVIEW

File: src/st/no3/46684.c [hunk 1]
```diff
@@ -1271,8 +1271,85 @@ INCLUDE_ASM("asm/us/st/no3/nonmatchings/46684", EntitySurfacingWater);
...
+            prim->y0 = y;
+            prim->r0 = 96;
+            prim->g0 = 96;
+            prim->b0 = 128;
+            prim->priority = self->zPriority + 2;
+            prim->blendMode = 0x33;
+            prim = prim->next;
+        }
+        //! FAKE:
+        ptr = &D_801838F4[params * 2];
```
Comment (sozud) [inline src/st/no3/46684.c:1322]: Did you try a struct here?

**Review by sozud — APPROVED** [review]

File: src/st/no3/46684.c (hunk 1 above)
Comment (sonicdcer) [reply src/st/no3/46684.c:1322]: Yes, i couldn't find a match, if you can find it i'd be grateful.

**Comment by sozud** [comment]
This is conflicted now

**Comment by sonicdcer** [comment]
I'll fix it in a few minutes

**Pull request merged**